🐛 Bug: [text-processor] type error in index.test.ts

### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/LearningTypeScript/projects/issues?q=is%3Aissue+label%3A%22type%3A+bug%22) and found none that matched my issue.
- [X] This is the appropriate issue form for the bug I would like to report.

### Expected

typescript watch mode to successfully run for text-processor project, but it throws an error in the index.test.ts file

### Actual

error TS2345: Argument of type '{ width: number; } | { align: string; width: number; }' is not assignable to parameter of type 'AlignmentOptions'.

### Impacted Project

projects/arrays/text-processor/src/index.test.ts

### Additional Info

_No response_

--- a/projects/arrays/text-processor/src/index.test.ts
+++ b/projects/arrays/text-processor/src/index.test.ts
@@ -22,7 +22,7 @@ describe(alignTexts, () => {
 		});
 	});
 
-	test.each([
+	const cases: [string[], index.AlignmentOptions, string[][]][] = [
 		[[""], { width: 0 }, [[""]]],
 		[["", ""], { width: 0 }, [[""], [""]]],
 		[[""], { width: 2 }, [["  "]]],
@@ -113,7 +113,9 @@ describe(alignTexts, () => {
 				["  abc", "  def", "  ghi"],
 			],
 		],
-	])("%j %o", (lines, options, aligned) => {
+	];
+
+	test.each(cases)("%j %o", (lines, options, aligned) => {
 		expect(alignTexts(lines, options)).toEqual(aligned);
 	});
 });

